Formula/portable-ruby.rb

```diff
@@ -1,17 +1,14 @@
 require File.expand_path("../../Abstract/portable-formula", __FILE__)
 
-class PortableRuby < PortableFormula
-  desc "Portable ruby"
+class PortableRubyAT23 < PortableFormula
```

Comment:
This should stay at `PortableRuby`, right?

Reply:
Oops, you're right - fixed.